src/helpers/Assets.php

```diff
@@ -915,20 +938,22 @@ public static function iconPath(string $extension): string
 
         $extLength = strlen($extension);
         if ($extLength <= 3) {
-            $textSize = '20';
+            $textSize = '19';
         } elseif ($extLength === 4) {
-            $textSize = '17';
+            $textSize = '16';
         } else {
-            if ($extLength > 5) {
-                $extension = substr($extension, 0, 4) . '…';
-            }
-            $textSize = '14';
-        }
-
-        $textNode = "<text x=\"50\" y=\"73\" text-anchor=\"middle\" font-family=\"sans-serif\" fill=\"#9aa5b1\" font-size=\"$textSize\">" . strtoupper($extension) . '</text>';
-        $svg = str_replace('<!-- EXT -->', $textNode, $svg);
+            $extension = substr($extension, 0, 3) . '…';
+            $textSize = '15';
+        }
```

Comment:
Updated font sizes based on visual tests with various 3, 4, 5+ extensions.
Since the ellipsis takes up a decent amount of space, I'm truncating now at 3 letters if over 4 chars.